doc/tutorials/setup_logical_partition_scenario_on_nuc7i7dnb.rst

```diff
@@ -0,0 +1,267 @@
+.. _setup_logical_partition_scenario_on_nuc7i7dnb:
+
+Guidelines of Setup Logical Partition Scenario on NUC7i7DNB
+############################################################
+
+ACRN hypervisor supports logical partition scenario, in which the User OS (such
+as Clear Linux) running in a pre-launched VM can bypass the ACRN hypervisor and
+directly access isolated PCI devices. The following guidelines provide step by
+step instructions on how to set up the ACRN hypervisor logical partition
+scenario on NUC7i7DNB boards running two pre-launched VMs. The logical partition
+scenario on NUC7i7DNB is shown in
+:numref:`logical_partition_scenario_on_nuc7i7dnb`.
+
+.. figure:: images/logical_partition_scenario_on_nuc7i7dnb.png
+   :align: center
+   :width: 400px
+   :name: logical_partition_scenario_on_nuc7i7dnb
+
+   The logical partition scenario on NUC7i7DNB
+
+Prerequisites
+*************
+
+* `NUC7i7DNB <https://ark.intel.com/content/www/us/en/ark/products/130394/intel-nuc-board-nuc7i7dnbe.html>`_
+* Connecting to the serial port, described in :ref:`getting-started-apl-nuc`
+* NVMe disk
+* SATA disk
+* Storage device with USB interface (such as USB Flash
+  or SATA disk connecting with USB3.0 SATA converter etc)
+* Disable "Intel Hyper Threading Technology" in the BIOS.
+
+  .. note::
+
+     To mitigate L1TF.
+
+     To avoid interference on the logical partition scenario between logical
+     cores.
+
+* In the logical partition scenario, two pre-launched VMs (running Clear Linux)
+  are started by the ACRN hypervisor. To set up the Linux root filesystems for
+  each VM, follow the `Install Clear Linux* OS on bare metal with live server
+  <https://clearlinux.org/documentation/clear-linux/get-started/bare-metal-install-server>`_
+  to install Clear Linux OS on a SATA disk and a storage device with USB
+  interface prior the setup, as the two pre-launched VMs will mount the
+  root file systems via the SATA controller and the USB controller respectively.
+
+Update kernel image and modules of Pre-launched VM
+**************************************************
+#. On your development workstation, clone the ACRN kernel source tree, and build
+   the Linux kernel image that will be used to boot the privileged VMs:
```

Comment:
"privliged" is a new term here.  Do we need to define it? You've been calling these "pre-launched VMs"

Reply:
Yes, thanks, should update it as "pre-launched VMs".

Reply:
will fix at next version.